test/backend.js

```diff
@@ -116,9 +116,9 @@ describe('Full walkthrough', function () {
   })
 
   after(function () {
-    for (let user of Object.values(users)) {
-      user.socket && user.socket.destroy({ timeout: 500 })
-    }
+    Object.keys(users).forEach((userKey) => {
+      users[userKey].socket && users[userKey].socket.destroy({ timeout: 500 })
+    });
```

Comment:
Changed it to the Object.keys() version only because it fixes the flow error even though using Object.values() is obviously simpler and cleaner. 
please refer to :
- https://github.com/facebook/flow/issues/2221
- https://github.com/facebook/flow/blob/master/lib/core.js#L65